actions/setup_e2e_tests.sh

```diff
@@ -41,29 +41,19 @@ if [[ -n "$RHTEST" ]]; then
         sudo service mongod restart
     fi
 
-    if [[ "$RHVERSION" -eq 6 ]]; then
-        # For RHEL/CentOS 6
-        sudo yum install -y python-pip jq epel-release
-    elif [[ "$RHVERSION" -eq 7 ]]; then
+    if [[ "$RHVERSION" -eq 7 ]]; then
         # For RHEL/CentOS 7
-        sudo yum install -y python-pip jq
+        sudo yum install -y python-pip jq bats
     else
         # For RHEL/CentOS 8 and above
-        sudo yum install -y python3-pip wget jq
+        sudo yum install -y python3-pip wget jq 
         PIP="pip3"
-    fi
-
-    # Remove bats-core if it already exists (this happens when test workflows
-    # are re-run on a server when tests are debugged)
-    if [[ -d bats-core ]]; then
+        # bats not available in epel for EL 8, Install from GitHub
```

Comment:
This is going to sound weird. You're right that bats is not available in EPEL for RHEL 8. But it is available, for some reason, via [npm](https://github.com/bats-core/bats-core#npm). So instead of installing from GitHub, let's install with npm:

```bash
npm install --global bats
```

Reply:
Odd - will fix